.github/workflows/ci.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Set up Go 1.12
+    - name: Set up Go 1.13
       uses: actions/setup-go@v1
       with:
-        go-version: 1.12
+        go-version: 1.13
```

Comment:
I think `GO111MODULE=on` below is not useful anymore, see https://golang.org/doc/go1.13#modules

Reply:
Good catch, thanks!